resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/DepsTestsSuite.scala

```diff
@@ -323,7 +384,7 @@ private[spark] trait DepsTestsSuite { k8sSuite: KubernetesSuite =>
     try {
       setupMinioStorage()
       val minioUrlStr = getServiceUrl(svcName)
-      createS3Bucket(ACCESS_KEY, SECRET_KEY, minioUrlStr)
+      createS3Bucket(minioUrlStr)
```

Comment:
This is related to the previous comment, https://github.com/apache/spark/pull/38376/files#r1069933779 . Please try to reduce the patch size by focusing on the required change .